File: acme.cashflow.domain/DTO/Product/Price/ValorProduto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace acme.cashflow.domain.DTO.Product.Price
{
    public class ValorProduto : AbstractEntity
    {
        public decimal Valor { get; set; }
        public Guid ProdutoId { get; set; }
        public Guid TipoValorProdutoId { get; set; }

        public Produto Produto { get; set; }
        public TipoValorProduto TipoValorProduto { get; set; }
    }
}

File: site/coreui-free-angular-admin-template-master/src/app/view-model/response/product/price/valor-produto.ts
import { Produto } from './../produto';
import { AbstractEntity } from "../../abstract-entity";
import { TipoValorProduto } from "./tipo-valor-produto";

export class ValorProduto extends AbstractEntity{
  public valor:number;
  public  produtoId : string;
  public  tipoValorProdutoId : string;

  public produto: Produto;
  public tipoValorProduto: TipoValorProduto;
}
